### PR TITLE
Bump to citus_enterprise 8.3.4

### PR DESCRIPTION
**Pull request by gurkanindibay**
Now nightly builds are failing for el since I needed to rollback to commit # 53f3d2df82c8b2a2949ccb4709af0dc3ea8e4762. After releasing all builds, .spec file will be the latest so it will be fixed

### CODE REVIEW

File: .github/workflows/update_package_properties.yml [hunk 1]
```diff
@@ -39,7 +39,7 @@ jobs:
         run: sudo apt-get update && sudo apt install libcurl4-openssl-dev libssl-dev python3-testresources
 
       - name: Clone tools branch
-        run: git clone -b v0.8.12 --depth=1  https://github.com/citusdata/tools.git tools
+        run: git clone -b v0.8.14 --depth=1  https://github.com/citusdata/tools.git tools
```
Comment (hanefi) [inline .github/workflows/update_package_properties.yml]:
```suggestion
        run: git clone -b v0.8.14 --depth=1 https://github.com/citusdata/tools.git tools
```

**Comment by hanefi** [comment]
PG10 did not yet have JIT support. Hence we do not generate `*.bc` files on builds. This causes some issues in tests such as the following:

```
    File not found by glob: /root/rpmbuild/BUILDROOT/citus-enterprise83_10-8.3.4.citus-1.el7.x86_64/usr/pgsql-10/lib/bitcode/citus*.bc
    File not found by glob: /root/rpmbuild/BUILDROOT/citus-enterprise83_10-8.3.4.citus-1.el7.x86_64/usr/pgsql-10/lib/bitcode/citus/*.bc
    File not found by glob: /root/rpmbuild/BUILDROOT/citus-enterprise83_10-8.3.4.citus-1.el7.x86_64/usr/pgsql-10/lib/bitcode/citus/*/*.bc
```

You can check the commit https://github.com/citusdata/packaging/commit/4c45c083429c5ca6be78bfb86c8c084cc2806095 that introduced the first release that had JIT support.

The root cause is that once we dropped support for PG10 in later versions of Citus, we also assumed that JIT support will always be there. This assumption causes headaches when doing backports to older versions.

**Comment by gurkanindibay** [comment]
> PG10 did not yet have JIT support. Hence we do not generate `*.bc` files on builds. This causes some issues in tests such as the following:
> 
> ```
>     File not found by glob: /root/rpmbuild/BUILDROOT/citus-enterprise83_10-8.3.4.citus-1.el7.x86_64/usr/pgsql-10/lib/bitcode/citus*.bc
>     File not found by glob: /root/rpmbuild/BUILDROOT/citus-enterprise83_10-8.3.4.citus-1.el7.x86_64/usr/pgsql-10/lib/bitcode/citus/*.bc
>     File not found by glob: /root/rpmbuild/BUILDROOT/citus-enterprise83_10-8.3.4.citus-1.el7.x86_64/usr/pgsql-10/lib/bitcode/citus/*/*.bc
> ```
> 
> You can check the commit [4c45c08](https://github.com/citusdata/packaging/commit/4c45c083429c5ca6be78bfb86c8c084cc2806095) that introduced the first release that had JIT support.
> 
> The root cause is that once we dropped support for PG10 in later versions of Citus, we also assumed that JIT support will always be there. This assumption causes headaches when doing backports to older versions.

I rollbacked to commit # https://github.com/citusdata/packaging/commit/53f3d2df82c8b2a2949ccb4709af0dc3ea8e4762.

**Comment by hanefi** [comment]
We decided to change the order we release the packages. `v8.3.4` will be the last item in a long list of versions that we will package.

**Pull request closed without merging**